### PR TITLE
[ONNX] Update all references of `onnxscript-preview` to `onnxscript`

### PR DESCRIPTION
**Pull request by justinchuby**

### CODE REVIEW

**Review by thiagocrepaldi — APPROVED** [review]

**Comment by thiagocrepaldi** [comment]
> _ No description provided. _

maybe add a reference to the ticket that approed the name change from onnxscript-preview to onnxscript? It would be teach people how to proceed with similar requests

File: test/onnx/test_fx_op_consistency.py [hunk 1]
```diff
@@ -201,16 +201,6 @@
         "amin", dtypes=(torch.int16, *onnx_test_common.BOOL_TYPES),
         reason=onnx_test_common.reason_dynamo_does_not_support("ReduceMin", "bool, int16")
     ),
-    xfail(
-        "arange",
```
Comment (BowenBao) [inline test/onnx/test_fx_op_consistency.py:205]: this is failing in CI though, is there an update to `arange` between the onnxscript version bump?

**Review by BowenBao — APPROVED** [review]

File: test/onnx/test_fx_op_consistency.py (hunk 1 above)
Comment (justinchuby) [reply test/onnx/test_fx_op_consistency.py:205]: There is but some tests that should pass w/ the new version failed - so I am trying to figure that out

**Comment by BowenBao** [comment]
Dup of #109139

File: test/onnx/test_fx_op_consistency.py [hunk 2]
```diff
@@ -554,7 +546,7 @@
         matcher=lambda sample: sample.kwargs.get("max_norm") is not None,
         reason="Torchlib does not support aten::embedding_renorm, emitted when 'max_norm' is not None",
     ),
-    xfail(
+    skip(
         "nn.functional.embedding_bag",
         matcher=lambda sample: sample.kwargs.get("padding_idx") is not None or True,
         reason=(
```
Comment (abock) [inline test/onnx/test_fx_op_consistency.py:552]: Refer to the issue with more details?

> `"See https://github.com/microsoft/onnxscript/issues/1056 for details."`

**Pull request closed without merging**